src/display/editor/tools.js

```diff
@@ -1154,6 +1156,21 @@ class AnnotationEditorUIManager {
     }
   }
 
+  enableWaiting(mustWait) {
```

Comment:
Let's prevent a future inconsistency here, if this method is ever called "incorrectly".
```suggestion
  enableWaiting(mustWait = false) {
```